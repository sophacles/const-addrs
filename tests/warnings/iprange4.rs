use const_addrs::iprange4;

fn main() {
    let _ = iprange4!("10.0.1.0", "10.0.0.0");

    // failure so output is compared
    let _ = iprange4!("10.0.10", "10.0.0.0");
}
