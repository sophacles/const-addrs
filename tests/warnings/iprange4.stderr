warning: This range will yeild no values: IP1 >= IP2
 --> tests/warnings/iprange4.rs:4:23
  |
4 |     let _ = iprange4!("10.0.1.0", "10.0.0.0");
  |                       ^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: This iterator requires the lower IP value first in it's arguments list.

error: invalid IP address syntax
 --> tests/warnings/iprange4.rs:7:23
  |
7 |     let _ = iprange4!("10.0.10", "10.0.0.0");
  |                       ^^^^^^^^^
